src/broker/broker_admin_pub.c

```diff
@@ -324,9 +326,17 @@ admin_start_cmd (T_BROKER_INFO * br_info, int br_num, int master_shm_id, bool ac
     }
   chdir (envvar_bindir_file (path, BROKER_PATH_MAX, ""));
 
+  if (gethostname (hostname, sizeof (hostname)) < 0)
+    {
+      fprintf (stderr, "gethostname error\n");
```

Comment:
fprintf (stderr, "gethostname error: cannot get local hostname\n");

Reply:
I will apply this. Thank you!